tasks/mina/install.rb

```diff
@@ -10,7 +12,7 @@
 
   outfile = './config/deploy.rb'
 
-  if File.exists?(outfile)
+  if File.exist?(outfile)
```

Comment:
`File.exists?` is [deprecated](https://ruby-doc.org/core-2.5.0/File.html#method-c-exists-3F) in favor of `File.exist?`